### PR TITLE
(tests) Add vars to pytest env

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -12,8 +12,10 @@ env =
...
...S0czClVMUzE1L3h2Q2N2T3p1TnhHSTRjNDVyaEVxUE8rMUlGODkvbEZ4aGRXZ1RTWmpiUGdvNXJiRGREb2IyVlpSMDAKbEZTS2dEbHNjV1YySEx6NzJQKzZWYlZVMEI4bVhWSzJhL3lCbERzRG9ZbzlJTDlCRTlVWTJrMzJEeHpvTjRregppbm5jL013L0ROZXB4c1luSWZaVU5JbFB5UUtCZ1FETFRKTTlqbWNXMFB0bktOOTlEeDdpRzQyREJMcHpQaXF1Ckh2ME5xbWZObXlZMHVKdkNKdG9DbVdMU28ycDM1d1BjSG9GTy9jYS9EdTEySURrWkpkRStiMjU0dDk3YUxkdDMKaUZkeUw4ZTI0OGFORWVqYVVud1lXRXBJbVlnRGVFTnJtT3RSSTF5clIwRjFTa0hoK29EZmdWL1cyQWJzTnJlQwpFd2lEeE5EYjV3S0JnUUNKSFpKZEFkNUR6QU1yWjFBUkVpY3VVWks4c0M5c0F5MnhxbjNpQkNsd1VqTkxzWG14CmFlQkJKaVpDcmNmVFNVTVZYRmVZYkNmczNQSTJjVlVQYUNLWFpVanRWNmtrTlc4V2c4eTBLdEZnUUNsb3htVDIKYjRwTnBFcFNMQzZwTElXWnhoMm93c1JTbnJvdkNLNzJQYVRqaGI2amtHUGlhdUxCbjk1ZjRJWFVDUUtCZ1FDWgptZVcxb0o0ckV3S2YwclhvMnovT1Bsc1JIc1Z3eFF1aU95K2UzNTFHTDI4U2Jsamd2RklEbkxpWnhYWnYyVStGClJGY1d3NDhpVTdqK2d5SWc2ZjNLV283eERwdUY1UmZvbkU3VnExd054dE1CU3dMUDBYcUpPa3dGeGxxSnBKZnIKOThIenIvQVlTT0lnS0ZiVzBsb0VoTTlRbUMvQzBTU2RWcHdjVXR3ckNRS0JnQytpdlBpRXZieVVram1mcmhTdwpFTGVvRHJMMVlTL251Qkk3NFN5S3NVTU5YL1JYVEJNNFpFUGZRRHlUdk9OUjRMME1ic20xZGQzcm1DMlFaSjZ0CmEyVENCL1BlQzVyOTUwRGlGVVhvempDVUxNbXpZbk5XVUJRUWZ3cmEzUTdEL2xtV1JWVDJOUXNEY29TcDFvM1QKUVd6QkV5ZnFDZTEySUQyZE1uZTBlbnVhCi0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K
```
Comment (lgarvey) [inline pytest.ini:15]: I'm surprised these are needed - not an issue adding them though.

File: pytest.ini [hunk 2]
```diff
@@ -12,8 +12,10 @@ env =
...
   SAML2_APPSTREAM_AWS_ROLE_ARN=
+  SAML2_QUICKSIGHT_AWS_ROLE_ARN=
```
Comment (lgarvey) [inline pytest.ini:18]: Doh. Thanks for adding this.

File: pytest.ini (hunk 1 above)
Comment (michalc) [reply pytest.ini:15]: Ah without, the test_alias_entry test seemed to be failing on me

File: pytest.ini (hunk 2 above)
Comment (michalc) [reply pytest.ini:18]: np

**Pull request merged**